src/style.css

```diff
@@ -1 +1,94 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
The `font-family` property should not include multiple fonts separated by commas within the `@font-face` rule. Instead, specify only the custom font name, like `Avenir`, and use fallback fonts in the `font-family` property of other CSS selectors.